package com.callisto.connect.exception;

import java.util.HashMap;

public class ExceptionHelper {

	private static boolean _isDebugging = true;

	public static boolean isDebugging() { return ExceptionHelper._isDebugging; }
	public static void setDebugging(boolean debug) { ExceptionHelper._isDebugging = debug; }
	
	public static final int ERROR_INVALID_PARAMETER = 1;
	
	private static HashMap<Integer, String> _dictionary = null;
	
	static {
		
		ExceptionHelper._dictionary = new HashMap<Integer, String>();
		
		// Initializations
	}
	
	public static String getErrorMessageByCode(int errorCode)
	{
		if (ExceptionHelper._dictionary.containsKey(errorCode) != true)
		{
			return null;
		}
		
		return ExceptionHelper._dictionary.get(errorCode);
	}

}
